File: CS/SpreadsheetWPFChartAPISamples/CodeUtils/CodeExampleGroupChildrenSelector.cs
using DevExpress.Xpf.Grid;
using SpreadsheetChartAPISamples;
using System;
using System.Collections;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SpreadsheetWPFChartAPISamples
{
    public class CodeExampleGroupChildrenSelector : IChildNodesSelector
    {
        IEnumerable IChildNodesSelector.SelectChildren(object item)
        {
            if (item is CodeExampleGroup)
                return ((CodeExampleGroup)item).Examples;
            return null;
        }
    }
}

File: VB/SpreadsheetWPFChartAPISamples/CodeUtils/CodeExampleGroupChildrenSelector.vb
Imports DevExpress.Xpf.Grid
Imports SpreadsheetChartAPISamples
Imports System.Collections

Namespace SpreadsheetWPFChartAPISamples

    Public Class CodeExampleGroupChildrenSelector
        Implements IChildNodesSelector

        Private Function SelectChildren(ByVal item As Object) As IEnumerable Implements IChildNodesSelector.SelectChildren
            If TypeOf item Is CodeExampleGroup Then Return CType(item, CodeExampleGroup).Examples
            Return Nothing
        End Function
    End Class
End Namespace
